scex-util/src/main/scala/com/avsystem/scex/util/function/NetFunctions.java

```diff
@@ -116,7 +116,7 @@ public static boolean isIps(final String ips) {
         }
 
         if (ips != null) {
-            for (String ip : ips.replaceAll(",\\s+", ",").split(",")) {
+            for (String ip : ips.trim().split("\\s*,\\s*")) {
```

Comment:
Wouldn't it be more natural to split on `','` and `trim()` each element independently? Also, be aware that `"".split(whatever) == Array("")`

Reply:
I have changed split with trim. Splitting empty string does not affect the way this function works since isIp function returns false when applied to the empty string so isIps also returns false for an empty string as it is expected